internal/server/v1/places/places.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 Google LLC
```

Comment:
Should we call this file "info.go", since this is already under "places".

Also I wonder should we use singular form for folder name like "place", since this is a general concept instead of REST resource

Reply:
Done.